src/makeCalculator.js

```diff
@@ -37,7 +37,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  function add(value) {
+    result += value;
+  }
+
+  function subtract(value) {
+    result -= value;
+  }
+
+  function multiply(value) {
+    result *= value;
+  }
+
+  function divide(value) {
+    result /= value;
```

Comment:
There is no check for divide by zero. Please handle this case because division by zero is undefined.